tests/commonTemplates_test.go

```diff
@@ -212,4 +215,63 @@ var _ = Describe("Common templates", func() {
 			table.Entry("[test_id:4770]golden image NS", &goldenImageNS),
 		)
 	})
+
+	Context("older templates update", func() {
+		It("should replace ownerReference with owner annotations for older templates", func() {
+			ssp := getSsp()
+			oldTpl := templatev1.Template{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      "test-old-template",
+					Namespace: ssp.Spec.CommonTemplates.Namespace,
+					Labels: map[string]string{
+						"template.kubevirt.io/version": "not-latest",
+						"template.kubevirt.io/type":    "base",
+					},
+					OwnerReferences: []v1.OwnerReference{{
+						APIVersion: "ssp.kubevirt.io/v1beta1",
+						Kind:       "SSP",
+						Name:       ssp.Name,
+						UID:        ssp.UID,
+					}},
+				},
+			}
+
+			err := apiClient.Create(ctx, &oldTpl)
```

Comment:
Could you delete the resource once the test is done?
So it doesn't collide with the defaults etc.

Reply:
Done